### ISSUE TITLE
experimental warnings under v5.22

### ISSUE THREAD

**Issue by RsrchBoy** [kan/p5-smart-options#8]
Using ```Smart::Options::Declare``` under v5.22, this is seen:

```
given is experimental at /home/cweyl/envperl5/lib/perl5/Smart/Options.pm line 347.
when is experimental at /home/cweyl/envperl5/lib/perl5/Smart/Options.pm line 348.
when is experimental at /home/cweyl/envperl5/lib/perl5/Smart/Options.pm line 352.
when is experimental at /home/cweyl/envperl5/lib/perl5/Smart/Options.pm line 355.
when is experimental at /home/cweyl/envperl5/lib/perl5/Smart/Options.pm line 362.
when is experimental at /home/cweyl/envperl5/lib/perl5/Smart/Options.pm line 369.
when is experimental at /home/cweyl/envperl5/lib/perl5/Smart/Options.pm line 376.
when is experimental at /home/cweyl/envperl5/lib/perl5/Smart/Options.pm line 380.
```

...this might be nice to quiet down :)

**Comment by kan** [comment]
I fixed that warnings at Smart::Options 0.053

https://github.com/kan/p5-smart-options/releases/tag/0.053

Please use Smart::Options 0.053 and later

**Comment by RsrchBoy** [comment]
Thanks for that -- I have no idea how I was still using 0.051, but I was.  Whoops :) *blush*
